regression/tests/lms/courses_enroll_unenroll.py

```diff
@@ -0,0 +1,87 @@
+import os
```

Comment:
Add module docstring (minor) and rename module to something with `test_` in the name.